src/packages/evasion.config

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+    <package id="CheckPlease.fireeye" />
+    <package id="DefenderCheck.fireeye" />
+    <package id="DotNetToJScript.fireeye" />
+    <package id="Invoke-CradleCrafter.fireeye" />
+    <package id="Invoke-DOSfuscation.fireeye" />
+    <package id="Invoke-Obfuscation.fireeye" />
+    <package id="Invoke-Phant0m.fireeye" />
+    <package id="PSAmsi.fireeye" />
+    <package id="PowerLessShell.fireeye" />
+    <package id="PowerShdll.fireeye" />
+    <package id="StarFighters.fireeye" />
+    <package id="SysWhispers.fireeye" />
+    <package id="demiguise.fireeye" />
+    <package id="nps.fireeye" />
+    <package id="pafishmacro.fireeye" />
```

Comment:
Since alphabetical sorting is intended to help humans, I think it makes sense to sort these (and all of the other package configs) in **non-case-sensitive** alphabetical order.  That would make life easier for someone who is trying to locate an item in the list.

Reply:
Thank you for suggesting improvements that lead me to learn a new thing about Vim 

Sort highlighted lines (i for case-insensitive): `'<,'>sort i`